CMakeLists.txt

```diff
@@ -279,7 +279,7 @@ include(CheckLibraryExists)
 find_package(LibUV REQUIRED)
 include_directories(SYSTEM ${LIBUV_INCLUDE_DIRS})
 
-find_package(Msgpack REQUIRED)
+find_package(Msgpack 1.0.0 REQUIRED)
```

Comment:
So if we want to bump the msgpack version, is this the only place that needs to be updated?